hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -278,11 +301,11 @@ public OMResponse submitRequest(OMRequest omRequest) throws ServiceException {
    */
   public OMResponse submitRequest(OMRequest omRequest,
       RaftClientRequest raftClientRequest) throws ServiceException {
-    RaftClientReply raftClientReply = submitRequestToRatis(raftClientRequest);
-    return processReply(omRequest, raftClientReply);
+    RaftClientReply raftClientReply = submitRequestToRatisImpl(raftClientRequest);
+    return createOmResponseImpl(omRequest, raftClientReply);
```

Comment:
This should use `createOmResponse`.

Reply:
@duongkame , ping, in case you don't see this.

Reply:
Hi @szetszwo , updated.